config/webpack/webpack.config.js

```diff
@@ -53,11 +53,14 @@ const defaultWebpackConfig = (env = {}, argv = {}) => {
                 return [
                   Autoprefixer({
```

Comment:
Once the `browserslist-config-terra` package is released, it will be added as a dependency and this list will be updated to use the list provided in `browserslist-config-terra`. It will be one-to-one so this is not a blocker.